src/main/java/net/swofty/gui/inventory/inventories/GUIEnchantmentTable.java

```diff
@@ -366,6 +380,14 @@ public ItemStack.Builder getItem(SkyBlockPlayer player) {
 
                 @Override
                 public void run(InventoryPreClickEvent e, SkyBlockPlayer player) {
+                    if(e.getClickedItem().getMaterial() == Material.GRAY_DYE)
+                        return;
+                    
+                    // TODO if someone know how to get the itemstack name, do it
```

Comment:
ItemStack#getDisplayName, and then run it through StringUtility.textFromComponent